dashboard/client/src/pages/job/hook/useJobProgress.ts

```diff
@@ -15,15 +15,23 @@ import { getJobProgress, getJobProgressByTaskName } from "../../../service/job";
 export const useJobProgress = (jobId?: string) => {
   const [msg, setMsg] = useState("Loading progress...");
   const [error, setError] = useState(false);
+  const [driverExits, setDriverExits] = useState(true);
   const [isRefreshing, setRefresh] = useState(true);
   const refreshRef = useRef(isRefreshing);
   const onSwitchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setRefresh(event.target.checked);
   };
   refreshRef.current = isRefreshing;
+  console.log(jobId);
   const { data: progress } = useSWR(
     "useJobProgress",
     async () => {
+      if (!jobId) {
+        // JobId is empty if there's no driver.
+        // In this case, we have no progress to report.
+        setDriverExits(false);
```

Comment:
@alanwguo not sure if it is the best way, but if we just return null, it prints the fake green bar which is pretty weird. So I added a new state. Lmk if you think the green bar is better. cc @scottsun94